test/clientconn_state_transition_test.go

```diff
@@ -616,3 +618,52 @@ func (s) TestConnectivityStateSubscriber(t *testing.T) {
 		}
 	}
 }
+
+// TestChannelStateWaitingForFirstResolverUpdate verifies the initial
+// state of the channel when a manual name resolver doesn't provide any updates.
+func (s) TestChannelStateWaitingForFirstResolverUpdate(t *testing.T) {
+	backend := stubserver.StartTestService(t, nil)
+	defer backend.Stop()
```

Comment:
Nothing uses backend -- delete it or eventually deliver the address and expect READY.

Reply:
Delivered the address as READY